addons/mail/static/src/core/common/message_reactions.scss

```diff
@@ -1,3 +1,20 @@
-.o-mail-MessageReaction.o-selfReacted {
-    background: mix($o-brand-primary, $o-view-background-color, 10%);
+.o-mail-MessageReaction {
+    &.o-selfReacted {
+        --border-opacity: 0.5;
+        background: mix($o-brand-primary, $o-view-background-color, 7.5%);
+
+        &:hover {
+            --border-opacity: 0.75;
+        }
+    }
+
+    &:not(.o-selfReacted) {
+        --border-opacity: 0.35;
+        border-color: rgba(var(--secondary-rgb), var(--border-opacity)) !important;
+
+        &:hover {
+            --border-opacity: 1;
+        }
+    }
```

Comment:
There is no border class if it's not `selfReacted`.

Or it could be included with the .btn class. if so, perhaps you have an extra `.border` in ` t-att-class`?

Reply:
Indeed the border is included in `.btn`. I can make it explicit if you prefer?

Reply:
Does that mean we can remove the border class? If so, we should have the `btn` class, border would be useless.

Reply:
Indeed, I removed `border` as this is implicit in `.btn`